analytics/terraform/datahub-on-eks/datahub-eks/values/datahub_values.yaml

```diff
@@ -0,0 +1,334 @@
+    requests:
+      cpu: 300m
+      memory: 256Mi
+
+datahub-mae-consumer:
+  image:
+    repository: linkedin/datahub-mae-consumer
+    tag: "v0.8.45"
+
+datahub-mce-consumer:
+  image:
+    repository: linkedin/datahub-mce-consumer
+    tag: "v0.8.45"
+
+datahub-ingestion-cron:
+  enabled: false
+  image:
+    repository: acryldata/datahub-ingestion
+    tag: "v0.8.45"
+
+elasticsearchSetupJob:
+  enabled: true
+  image:
+    repository: linkedin/datahub-elasticsearch-setup
+    tag: "v0.8.45"
+  podSecurityContext:
+    fsGroup: 1000
+  securityContext:
+    runAsUser: 1000
+  podAnnotations: {}
+  extraEnvs:
+      - name: USE_AWS_ELASTICSEARCH
+        value: "true"
+  
+kafkaSetupJob:
+  enabled: true
+  image:
+    repository: linkedin/datahub-kafka-setup
+    tag: "v0.8.45"
+  podSecurityContext:
+    fsGroup: 1000
+  securityContext:
+    runAsUser: 1000
+  podAnnotations: {}
+
+mysqlSetupJob:
+  enabled: true
+  image:
+    repository: acryldata/datahub-mysql-setup
+    tag: "v0.8.45"
+  podSecurityContext:
+    fsGroup: 1000
+  securityContext:
+    runAsUser: 1000
+  podAnnotations: {}
+
+postgresqlSetupJob:
+  enabled: false
+  image:
+    repository: acryldata/datahub-postgres-setup
+    tag: "v0.8.45"
+  podSecurityContext:
+    fsGroup: 1000
+  securityContext:
+    runAsUser: 1000
+  podAnnotations: {}
+
+
+datahubUpgrade:
+  enabled: true
+  image:
+    repository: acryldata/datahub-upgrade
+    # tag: "v0.10.0"  # defaults to .global.datahub.version
+  batchSize: 1000
+  batchDelayMs: 100
+  noCodeDataMigration:
+    sqlDbType: "MYSQL"
+    # sqlDbType: "POSTGRES"
+  podSecurityContext: {}
+    # fsGroup: 1000
+  securityContext: {}
+    # runAsUser: 1000
+  podAnnotations: {}
+  # Add extra sidecar containers to job pod
+  extraSidecars: []
+    # - name: my-image-name
+    #   image: my-image
+    #   imagePullPolicy: Always
+  cleanupJob:
+    # Add extra sidecar containers to job pod
+    extraSidecars: []
+      # - name: my-image-name
+      #   image: my-image
+      #   imagePullPolicy: Always
+  restoreIndices:
+    resources:
+      limits:
+        cpu: 500m
+        memory: 512Mi
+      requests:
+        cpu: 300m
+        memory: 256Mi
+    # Add extra sidecar containers to job pod
+    extraSidecars: []
+      # - name: my-image-name
+      #   image: my-image
+      #   imagePullPolicy: Always
+
+## Runs system update processes
+## Includes: Elasticsearch Indices Creation/Reindex (See global.elasticsearch.index for additional configuration)
+datahubSystemUpdate:
+  image:
+    repository: acryldata/datahub-upgrade
+    # tag:
+  podSecurityContext: {}
+    # fsGroup: 1000
+  securityContext: {}
+    # runAsUser: 1000
+  podAnnotations: {}
+  resources:
+    limits:
+      cpu: 500m
+      memory: 512Mi
+    requests:
+      cpu: 300m
+      memory: 256Mi
+  # Add extra sidecar containers to job pod
+  extraSidecars: []
+    # - name: my-image-name
+    #   image: my-image
+    #   imagePullPolicy: Always
+
```

Comment:
Write a note on why you commented this section. Is this required?

Reply:
example / attributes for items in extraSidecars[]